Makefile

```diff
@@ -49,7 +49,9 @@ config:
 	@echo "// Automatically generated by \`make config BUILDDIR=$(BUILDDIR)\`, do not edit." >> llvm_config_$(GOOS).go
 	@echo "" >> llvm_config_$(GOOS).go
 	@echo "// #cgo CPPFLAGS: $(shell $(CONFIG) --cppflags)" >> llvm_config_$(GOOS).go
-	@if [ $(VERSION_MAJOR) -gt 9 ]; then \
+	@if [ $(VERSION_MAJOR) -gt 15 ]; then \
+	echo "// #cgo CXXFLAGS: -std=c++17" >> llvm_config_$(GOOS).go; \
+	elif [ $(VERSION_MAJOR) -gt 9 ]; then \
```

Comment:
I have removed this file in 4ed65198e00027da5ff1edfd231a2f160ea09f1d.

Reply:
Oh, I actually made some use of that https://src.fedoraproject.org/rpms/golang-tinygo-x-llvm/blob/rawhide/f/golang-tinygo-x-llvm.spec#_71; I guess I'll have to put it together manually.

Reply:
Oh, hmmm.
Why are you regenerating those files instead of using the existing config files?